packages/theme/src/cli/services/pull.ts

```diff
@@ -1,12 +1,18 @@
+/* eslint-disable tsdoc/syntax */
```

Comment:
Our `eslint` rules don't the period in `flags.path` 

In most of the repo, it seems like we don't outline the object properties in the JSDoc comments [example](https://github.com/Shopify/cli/blob/606087c80e07aff74306dab99c442bc97e76a1f5/packages/app/src/cli/models/app/loader.ts#L976-L984)

Since this is a complex object that consists of flags though, I am ignoring these rules.
Open to discussion!